src/tools/tidy/src/main.rs

```diff
@@ -71,6 +71,11 @@ fn filter_dirs(path: &Path) -> bool {
     skip.iter().any(|p| path.ends_with(p))
 }
 
+fn walk_many(paths: &Vec<&Path>, skip: &mut FnMut(&Path) -> bool, f: &mut FnMut(&Path)) {
```

Comment:
I've tried  `&[Path]` originally as well, but it gave me an error that Path is not sized ([doc](https://doc.rust-lang.org/stable/std/path/struct.Path.html) says this too). And when I did `&[&Path]` it gave me a type mismatch, not being able to coerce the `&Vec` to the slice.